book/go/api/api.md

```diff
@@ -184,13 +184,13 @@ An example that partitions words for a word-count based on their first character
 ```go
 type WordPartitionFunction struct {}
 
-func (wpf *WordPartitionFunction) Partition (data interface{}) uint64 {
+func (wpf *WordPartitionFunction) Partition (data interface{}) []byte {
```

Comment:
Take a look at the code in `examples/go/word_count/go/src/word_count/word_count.go` to make sure this matches what we have there.